packages/components/select-v2/__tests__/select.spec.ts

```diff
@@ -526,22 +534,24 @@ describe('Select', () => {
         },
       })
       await nextTick
```

Comment:
I think you might need to fix all the statement of 
```diff
--- await nextTick
+++ await nextTick()
```

Reply:
> I think you might need to fix all the statement of
> 
> ```diff
> --- await nextTick
> +++ await nextTick()
> ```

Oh my God, I'm sorry, I got a lot of mistakes